keylime/ima.py

```diff
@@ -229,6 +234,13 @@ def process_measurement_list(lines, lists=None, m2w=None):
 
         errs[3] += 1
 
+    # check PCR value has been found
+    if not found_pcr:
+        logger.error(
+            "IMA measurement list expected pcr value %s does not match TPM PCR %s"
+            %(codecs.encode(runninghash, 'hex').decode('utf-8'), pcrval))
```

Comment:
on second thought, this error message looks off, as there is no "expected pcr value".

```suggestion
        logger.error("IMA measurement list does not match TPM PCR %s" % pcrval)
```